### PR TITLE
[openwebnet] Fix README for Sitemap AUX configuration  to the correct mapping

### LINKED ISSUE #15254: Fix sitemap'ss configuration suggested on README for the aux  item that does not work.
# Description

In  the past a user pointed out that the aux item/channel did not work with sitemaps:

https://community.openhab.org/t/openwebnet-bticino-legrand-binding-beta-4-0-0-0-4-1-0-0/129775/30?u=giovanni_fabiani

Finally a solution has been found: the item in the Sitemap   should configured as: 

_Selection item=iAlarm_Control            label="Arm/Disarm alarm"   icon="shield" mappings=[OFF="Disarm", ON="Arm"]_  

The rest remains unchanged

# Test

Tested on Bticino Alarm unit 3486

### PR DESCRIPTION
**Pull request by fabgio**
# Description

Current sitemap configuration suggested on README does not work with  AUX items. in the past a user pointed out that the aux had a problem  with sitemaps. I confirm that   when the item is set as Switch on sitemap it does not work, in fact programmatically the aux command is defined as StringType.

https://community.openhab.org/t/openwebnet-bticino-legrand-binding-beta-4-0-0-0-4-1-0-0/129775/30?u=giovanni_fabiani

Finally a solution has been found 

The rest remains unchanged

 # Testing

Tested on Bticino Alarm unit 3486
closes #15254

### CODE REVIEW

File: bundles/org.openhab.binding.openwebnet/README.md [hunk 1]
```diff
@@ -456,13 +456,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
     
     Frame label="Alarm"
     {
-         Switch  item=iAlarm_System_State       label="Alarm state"
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/README.md:459]: Why did you remove this element ?

File: bundles/org.openhab.binding.openwebnet/README.md [hunk 2]
```diff
@@ -456,13 +456,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
...
-         Switch  item=iAlarm_Control            label="Arm/Disarm alarm"   icon="shield"
-         Switch  item=iAlarm_System_Armed       label="Armed"              icon="shield"
-         Switch  item=iAlarm_System_Network     label="Network"            icon="network"
-         Default item=iAlarm_System_Battery     label="Battery"            icon="battery"
-         Switch  item=iAlarm_Zone_3_State       label="Zone 3 state"
-         Default item=iAlarm_Zone_3_Alarm       label="Zone 3 alarm"       icon="siren"
+         Selection item=iAlarm_Control            label="Arm/Disarm alarm"   icon="shield" mappings=[OFF="Disarm", ON="Arm"]
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/README.md:459]: You can keep a Switch element, of course keeping the mappings you added. This will show two buttons, "Disarm" and "Arm". That is very probably better than a selection in this case.

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
@mvalla : please also review.

File: bundles/org.openhab.binding.openwebnet/README.md (hunk 2 above)
Comment (fabgio) [reply bundles/org.openhab.binding.openwebnet/README.md:459]: Ok I will try as  you suggested

Comment (fabgio) [reply bundles/org.openhab.binding.openwebnet/README.md]: Ok it works

File: bundles/org.openhab.binding.openwebnet/README.md (hunk 1 above)
Comment (fabgio) [reply bundles/org.openhab.binding.openwebnet/README.md:459]: I am not able to get of those unwanted spaces! any idea?

Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/README.md:459]: can you just copy+paste the line from a previus version/commit so that you restore the line as it was before?

Comment (fabgio) [reply bundles/org.openhab.binding.openwebnet/README.md:459]: Ok now only the line number 460 looks changed

**Comment by fabgio** [comment]
@lolodomo @mvalla Tjank you so much!
